src/components/LoginPage.tsx

```diff
@@ -0,0 +1,40 @@
+import { useFirebase } from 'react-redux-firebase'
+import { useNavigate } from 'react-router-dom'
+import { AppBar, Button, Toolbar, Typography } from '@mui/material'
+
+export const LoginPage = () => {
+  const firebase = useFirebase()
+  const navigate = useNavigate()
+
+  const signInWithGoogle = () => {
+    firebase
+      .login({
+        provider: 'google',
+        type: 'popup',
+      })
+      .then(() => {
+        navigate('/dashboard')
+      })
+  }
+  return (
+    <div>
+      <AppBar position="static">
+        <Toolbar>
+          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+            Trollo LoginPage
+          </Typography>
+          <Button
+            color="inherit"
+            onClick={(event) => {
+              event.preventDefault()
+              signInWithGoogle()
+            }}
```

Comment:
You could move `event.preventDefault()` inside `signInWithGoogle` function, but do you need it anyway? I'm not sure 🤔

Reply:
I think it can be removed